[FEAT-12-13] Fix type CommnadsScheduler, Spefic Commands #12  #13 (#15)
taecontrol/larastats-docs: docs/ssl-certificate-check/ssl-certificate-check-commands.md

## CheckSslCertificateCommand

To add the ``CheckSslCertificateCommand`` to your Laravel application, follow these steps:

> Open the ``app/Console/Kernel.php`` file in your Laravel project.
> Now we import the ``CheckSslCertificateCommand`` class:

Reviewer: ```suggestion
Open `app/Console/Kernel.php` file in your Laravel project and import the `CheckSslCertificateCommand` class:
```
